### ISSUE TITLE
Breadcrumb links only missing non-color designator (A11y)

### ISSUE THREAD

**Issue by Anschur** [OfficeDev/office-ui-fabric-react#8356]
### Environment Information 
 
- __Package version(s)__: (fill this out) 
- __Browser and OS versions__: (fill this out if relevant)

### Please provide a reproduction of the bug in a codepen:
 

I do not think links in the breadcrumb are up to the accessibility requirements. In this [article ](https://webaim.org/techniques/hypertext/link_text) it states that "The link must present a "non-color designator" (typically the introduction of the underline) on both mouse hover and keyboard focus." The breadcrumb links does not do this currently.

1. Go to https://developer.microsoft.com/en-us/fabric#/components/breadcrumb
2. Hover on a breadcrumb 

 
#### Actual behavior:
 
Background color changes slightly 
 
#### Expected behavior:
 
Links to have a underline

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No)
 
Requested priority: (Blocking, High, Normal, Low) 
 
Products/sites affected: (if applicable)

**Comment by khmakoto** [comment]
@micahgodbolt @jdhuntington What do you think about this? Should we add the underline designator?

I can do it, but just want to double-check first with you.

**Comment by micahgodbolt** [comment]
This is a large chunk of our controls TBH. But when it comes down to it, it's our designer's decisions on how we represent these various states, and our job is to follow that spec. 

@betrue-final-final do you have any feedback on if we have decided to intentionally excuse this 'non color designator'? Is this something you're working on in future revs/fluent?

**Comment by micahgodbolt** [comment]
also, adding a background could be considered a 'non color designator'. Adding a background might actually be more obvious than an underline. Again, we follow design's guidance.

**Comment by micahgodbolt** [comment]
@khmakoto as for adding the underline, definitely not without design approval. My guess is that we'll close this out as 'by design'. If there actually is a problem with the accessability of this control, that discussion needs to start with design. It's our job to implement it.

**Comment by khmakoto** [comment]
I'll soft close this for now and leave it like that unless we get instructions from design to address this.

**Comment by Anschur** [comment]
I had a chat with Ben about this, and he said that the background square qualifies as a non-color designation, but the current contrast is too low. So he is working on finding a new color if I understood him correctly.
